WordPress/Classes/Utility/WPWebViewController.m

```diff
@@ -440,10 +436,10 @@ - (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error
 - (void)displayLoadError:(NSError *)error
 {
     if (![ReachabilityUtils isInternetReachable]) {
-        [ReachabilityUtils showAlertNoInternetConnection];
+        [ReachabilityUtils showNoInternetConnectionNotice];
         [self reloadWhenConnectionRestored];
     } else {
-        [WPError showAlertWithTitle: NSLocalizedString(@"Error", @"Generic error alert title") message: error.localizedDescription];
+        [ReachabilityUtils showNetworkingErrorNoticeWithMessage:error.localizedDescription];
```

Comment:
I'm curious, if the error is not caused by reachability issue, why are we letting the reachabilityUtils handle it? For example a 404 is not a networking error so maybe the naming of the method (showNetworkingErrorNoticeWithMessage) is not accurate either?

Reply:
Good point. I wasn't actually sure where to place this method. First, I wished that `WPError` and `ReachabilityUtils` can be merged. They both have related functionality (dispatching Alerts/Notices) and I was concerned about splitting them up everywhere where they're hard to find and cause possible duplication in the future. So I thought that all networking-related errors should be in `ReachabilityUtils`. 

I wonder though. Maybe both `showNoInternetConnectionNotice` and `showNetworkingErrorNotice` should be in `WPError` instead? What do you suggest?